<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>504</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>952</width>
    <height>504</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>952</width>
    <height>504</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Координата X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qlineEdit_X">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qlineEdit_Y">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Координата Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qPushButton_AddPoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить точку в список</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qPushButton_ClearPointList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>251</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить список точек</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qComboBox_Interpolation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="qPushButton_ApplyInterpolation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>251</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Применить вариант интерполяции</string>
     </property>
    </widget>
    <widget class="QListWidget" name="qListWidget_Points">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>251</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qlineEdit_Step">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг интерполяции</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="qLabel_Map">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>650</width>
      <height>490</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>490</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>490</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
